docs/opcodes/A4.mdx

```diff
@@ -11,7 +11,7 @@ This instruction has no effect on the EVM state. See [here](https://ethereum.org
 
 ## Stack input
 
-0. `topic`: byte topic in the [memory](/about) in bytes.
+0. `offset`: byte topic in the [memory](/about) in bytes.
```

Comment:
Thanks for identifying, there is actually another typo in byte "topic" which should be offset as well

Reply:
Oh, I see it. Great eyes.

Reply:
changed my commit to include the 2nd typo as well.